docs/root/start/sandboxes/gzip.rst

```diff
@@ -83,3 +83,13 @@ Now, use ``curl`` to make a request for the compressed statistics:
 
     $ curl -si -H "Accept-Encoding: gzip" localhost:9902/stats/prometheus | grep "content-encoding"
     content-encoding: gzip
+
+.. seealso::
+   :ref:`Gzip Api <envoy_v3_api_field_extensions.filters.http.gzip.v3.Gzip.compressor>`
+      API and configuration reference for Envoy's gzip compression.
+
+   :ref:`Gzip configuration <config_http_filters_gzip>`.
+      Reference documentation for Envoy's gzip compression.
```

Comment:
These two are deprecated. Better reference the generic compressor filter configured with gzip as a compression library.

Reply:
I have updated it.